conf/main/logback.xml

```diff
@@ -62,11 +62,11 @@
         <appender-ref ref="${jetty.log.appender:-MAIN}"/>
     </logger>
 
-    <logger name="ai.grakn" level="${grakn.log.level}">
+    <logger name="ai.grakn" level="${grakn.log.level:-INFO}">
```

Comment:
@sheldonkhall Ah sorry I misunderstood. I did not realise this was setting the default to INFO. I thought it was setting the name of the log file for some reason. No complaints about this then.